### DISCUSSION TITLE
CPO - POLE FIGURES - GRID DATA - Create list of pole figures with hkl & uvw 100,010,001

### DISCUSSION THREAD

**Discussion by Stefaniacorvo** [mtex-toolbox/mtex#649] [Ask Anything]
Hi!
I need to plot pole figures for mineral of cubic (garnet), trigonal (quartz) and monoclinic (plagioclase or biotite) crystal symmetry.
Could you help me and confirm me if the following script is correct, please? 
```matlab
%%
% PF1qtz_100 = Miller(1,0,0,Quartz_CS,'uvw');
% PF2qtz_010 = Miller(0,1,0,Quartz_CS,'uvw');
% PF3qtz_001 = Miller(0,0,1,Quartz_CS,'uvw');
% PFs_qtz = {PF1qtz_100,PF2qtz_010,PF3qtz_001};
% %
PF1grt_100 = Miller(1,0,0,Garnet_CS,'uvw');
PF2grt_010 = Miller(1,1,0,Garnet_CS,'uvw');
PF3grt_001 = Miller(1,1,1,Garnet_CS,'uvw');
PFs_grt = {PF1grt_100,PF2grt_010,PF3grt_001};
% %
PF1bt_100 = Miller(1,0,0,Biotite_CS,'uvw');
PF2bt_010 = Miller(0,1,0,Biotite_CS,'uvw');
PF3bt_001 = Miller(0,0,1,Biotite_CS,'uvw');
PFs_bt = {PF1bt_100,PF2bt_010,PF3bt_001};
% %
PF1plag_100 = Miller(1,0,0,Plagioclase_CS,'uvw');
PF2plag_010 = Miller(0,1,0,Plagioclase_CS,'hkl');
PF3plag_001 = Miller(0,0,1,Plagioclase_CS,'hkl');
PFs_plag = {PF1plag_100,PF2plag_010,PF3plag_001}
%%
```
I'm not sure about the correct way I put 'uvw' or I need to put (hkl)? I know that this depend of the crystal symmetry, but I don't know very well how it works in a Script.
I'm using the version MTEX5.1.1.
Thanks,
Stefania

**Comment by kilir** [comment]
Hi,
'uvw' are crystal directions, 'hkl' are planes. I'm not really sure what you like to accomplish, but so far, your definitions of Millers look good. Many might consider for trigonal phases such as quartz the 4-digit Miller-Bravais (hkil) or Weber/Schweizer symbol (uvtw).
Cheers,
Rüdiger
